test/initialization.test.ts

```diff
@@ -32,13 +32,14 @@ suite('Initialization Tests', () => {
         Telemetry.disable();
     });
 
-    test('Connection manager is initialized properly', () => {
+    test('Connection manager is initialized properly', (done) => {
         // Wait for the extension to activate
-        ensureExtensionIsActive().then(() => {
+        ensureExtensionIsActive().then(async () => {
```

Comment:
If you're changing this to async anyways you might as well make the whole test function async and then can remove the done callback (same with the other similar tests)